internal/store/migrations.go

```diff
@@ -1238,6 +1238,79 @@ var migrations = []migration{
 			return err
 		}
 
+		return nil
+	}},
+	{semver.MustParse("0.26.0"), semver.MustParse("0.27.0"), func(e execer) error {
+		// 1. Add InstallationDBMigrationOperation table.
+		// 2. Add column MigratedInstallationsRaw to MultitenantDatabase table.
+		_, err := e.Exec(`
+			CREATE TABLE InstallationDBMigrationOperation (
+				ID TEXT PRIMARY KEY,
+				InstallationID TEXT NOT NULL,
+				RequestAt BIGINT NOT NULL,
+				State TEXT NOT NULL,
+				SourceDatabase TEXT NOT NULL,
+				DestinationDatabase TEXT NOT NULL,
+				SourceMultiTenantRaw BYTEA NULL,
+				DestinationMultiTenantRaw BYTEA NULL,
+				BackupID TEXT NOT NULL,
+				InstallationDBRestorationOperationID TEXT NOT NULL,
+				CompleteAt BIGINT NOT NULL,
+				DeleteAt BIGINT NOT NULL,
+				LockAcquiredBy TEXT NULL,
+				LockAcquiredAt BIGINT NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = e.Exec(`ALTER TABLE MultitenantDatabase RENAME TO MultitenantDatabaseTemp;`)
```

Comment:
I believe it's possible to just alter this table in place by adding the new column. I don't think you have to rename to a temp table and copy over. This pattern appears in a few places in our migrations but I believe it was erroneously believed at one time that sqlite won't allow you to add a column with ALTER TABLE and so the pattern has proliferated, but I think you can just do ALTER TABLE ADD COLUMN and then skip INSERT INTO on line 1291

It's possible that I'm wrong about this, as I'm pretty sure I've pointed this out in @gabrieljackson 's code before, and now I'm second-guessing my memory about the outcome, but if it's possible to just use ALTER TABLE to add the column without copying the data around I'd prefer that here

Reply:
Thanks for checking on that. Yes, you are absolutely right that sqlite allows adding column with ALTER TABLE. The point here is that I want new column to be the same as `InstallationsRaw` as they store data in the same form. This means that the column should be `BYTEA NOT NULL`. This means we can only add column with `DEFAULT` constraint as all existing rows will have to get some value (it cannot be NULL). I do not want to stick with DEFAULT constraint on the column tho so that it we can keep it exactly the same as `InstallationsRaw` and possibly avoid some surprises in the future. The issue with sqlite arises here as it does not support dropping the constraint on the column.

The SQL would look something like that and it does work with Postgres but unfortunately not with sqlite:
```sql
ALTER TABLE MultitenantDatabase ADD COLUMN MigratedInstallationsRaw BYTEA NOT NULL DEFAULT '[]';
ALTER TABLE MultitenantDatabase ALTER COLUMN MigratedInstallationsRaw DROP DEFAULT; -- SQLite does not supprot this one
```